### PR TITLE
Add Node API

### LINKED ISSUE #363: update node api docs to include fee parameter for transaction broadcasts
https://github.com/celestiaorg/celestia-node/pull/1484 introduces the following changes to the gateway requests:

All transaction endpoints now have a `"fee"` key needed in the JSON request. An example value for this can be `2000`. This includes the following endpoints:

- [x] SubmitPayForData
- [x] Transfer
- [ ] Delegate
- [ ] BeginRedelegate
- [ ] Undelegate
- [ ] CancelUnbondingDelegation

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

Resolves #363 

- [x]  https://docs.celestia.org/developers/node-tutorial/#submit-a-pfd-transaction needs a disclaimer that this endpoint will be deprecated in favor of the RPC (or we just already swap it out with the RPC CURL)
- [x] https://docs.celestia.org/developers/node-api/ Endpoints to be deprecated: `transfer` and `submit_pfd`
- [x] write markdown file with structure of Header module @jcstein 
- [x] Add react page or components, implementing similar to https://github.com/celestiaorg/node-rpc-docs
- [ ] isolate tailwind styling postCSS to only the .tsx component (page) 
- [ ] solve issue with parsing error on some of the methods

## Checklist

- [x] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

**Comment by jcstein** [comment]
do you think you can split up the prototype's files in the `/pages` directory? the same setup that you had in your other version maybe?

**Comment by jcstein** [comment]
https://user-images.githubusercontent.com/46639943/220795005-95547ad1-a14f-44b5-9a76-9075ca7ad274.mp4

**Comment by jcstein** [comment]
https://www.loom.com/share/aa79d05f9d7342b7a6f0bfd8f762f148

update!

File: src/css/tailwind.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.tailwind-layout {
```
Comment (lzrscg) [inline src/css/tailwind.css:1]: I would try adding `all: unset;` in here

btw I am having trouble running this repo. I do `yarn start` which works, but localhost:3000 hangs forever

File: src/pages/node-api-docs.tsx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+      });
+    }
+  }
+  return methodsByPackage;
+}
+
+export default function Hello() {
+  // useTailwindLayout();
+  return (
+      <div className={styles}>
```
[the remark this answers is not in the document]
Comment (jcstein) [reply src/pages/node-api-docs.tsx:61]: yep! currently attempting to use the styles imported above in `import styles from "../css/tailwind.css"`

File: src/css/tailwind.css (hunk 1 above)
Comment (jcstein) [reply src/css/tailwind.css:1]: will try this! hmm. feels like a "did you power it on and off" question, but did you run `yarn` first?

Comment (lzrscg) [reply src/css/tailwind.css:1]: ya I did try that :/ its running smoothly, but not loading

**Comment by jcstein** [comment]
it's working right for [this component on this page](https://docs.celestia.org/pr-preview/pr-445/developers/node-api/), but still applying to rest of components across site

**Comment by jcstein** [comment]
https://user-images.githubusercontent.com/46639943/221323248-b284e481-fca6-49fc-bf7b-651e2de5172a.mp4

**Comment by jcstein** [comment]
I'm closing this in order to reduce potential confusion between the 2 PRs

**Pull request closed without merging**